app/controllers/posts_controller.rb

```diff
@@ -24,17 +24,17 @@ def edit
 
   def create
     @post = Post.new(post_params)
-    flash[:notice] = "Successfully created the post." if @post.save
+    flash[:notice] = 'Successfully created the post.' if @post.save
     respond_with @post
   end
 
   def update
-    flash[:notice] = "Successfully updated the post." if @post.update(post_params)
+    flash[:notice] = 'Successfully updated the post.' if @post.update(post_params)
```

Comment:
Line is too long. [82/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.